### PR TITLE
new recipe: ipyrad (please review and merge)

### PR DESCRIPTION
**Pull request by eaton-lab**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by eaton-lab** [comment]
@bioconda/core Please review and merge, and I would like to be added to the bioconda project to continue updates. Thanks.

**Comment by dpryan79** [comment]
@bioconda-bot merge

File: recipes/ipyrad/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  version: {{version}}
+
+source:
+  url: https://github.com/eaton-lab/{{name}}/archive/{{version}}.tar.gz
+  sha256: {{sha256}}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  noarch: generic
```
Comment (epruesse) [inline recipes/ipyrad/meta.yaml]: @dpryan79 Why `generic` and not `python`? Does this install things outside of `site-libraries`?

Comment (dpryan79) [reply recipes/ipyrad/meta.yaml]: Oh, I missed that!

Comment (eaton-lab) [reply recipes/ipyrad/meta.yaml]: Hey all, I did try with `noarch: python` but was having problems with build numbers. Sorry for the confusion, I didn't realize I could re-open this pull request after it was closed so I started a new one: #15791

Comment (epruesse) [reply recipes/ipyrad/meta.yaml]: You can re-open merged PRs??? Ugh... really?

**Pull request merged**